ArchUnitNET/Domain/PlantUml/Export/PlantUmlDependency.cs

```diff
@@ -67,6 +175,14 @@ public override int GetHashCode()
     public enum DependencyType
     {
         OneToOne,
-        OneToMany
+        OneToMany,
+        OneToPackage,
+        PackageToOne,
+        PackageToPackage,
+        OneToOneIfSimilarNamespace,
```

Comment:
My impression from reading the code is that this should be called `OneToOneIfSameParentNamespace`, same for the next one. Do you agree?